AudioSystem/AudioSystem.py

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Any particular reason this is done in Python?

Reply:
Cross compatibility. Can't find a c# lib that will allow the manipulation of the audio easily and is cross platform.